community_images/common/dockertags.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# This scripts fetchs all dockertags for a given repo
+
```

Comment:
why is this script changes/added?

Reply:
Seems like it existed in the main branch when I created a branch. Removing it.